### PR TITLE
Add string for query search

### PR DESCRIPTION
**Pull request by domenc**
Add string "Query result for" on search page - issue #86

### CODE REVIEW

File: piplmesh/frontend/templates/search.html [hunk 1]
```diff
@@ -5,6 +5,9 @@
 {% block title %}{% trans "Search" %}{% endblock %}
 
 {% block content %}
+    {% if "q" in request.GET %}
+    <div id="google_query">{% trans "Query results for:" %}&nbsp;<b>{{ request.GET.q }}</b></div>
```
Comment (mitar) [inline piplmesh/frontend/templates/search.html]: Indent. You do not have to use non-braking space. Just put a normal space there and this is it. Don't use `<b>`, use span with class and style that with CSS.

**Comment by mitar** [comment]
BTW, with "Past search query string should be visible on result page." I though that in the search input box, past query string should be visible. So when you search and come to results, search input box should be empty.

**Comment by domenc** [comment]
ok :)

File: piplmesh/frontend/templates/search.html [hunk 2]
```diff
@@ -5,6 +5,9 @@
 {% block title %}{% trans "Search" %}{% endblock %}
 
 {% block content %}
+    {% if "q" in request.GET %}
+    <div id="google_query">{% trans "Query results for:" %} <span class="google_query_text">{{ request.GET.q }}</span></div>
```
Comment (mitar) [inline piplmesh/frontend/templates/search.html]: Indentation! Inside of `if` you should indent once.

Use only _Results for:_ as a string. :-)

**Comment by domenc** [comment]
'BTW, with "Past search query string should be visible on result page." I though that in the search input box, past query string should be visible. So when you search and come to results, search input box should be empty.'

Javascript not working -> $('[name='search']').val(query)

And image left of the input box. Why? Isn't go taht image is inside input box?

**Comment by mitar** [comment]
Why JavaScript for that? I think you can do it without? In template rendering?

Image on the left? Just check how Firefox Google search looks like. Do same.

**Comment by domenc** [comment]
Because, JS create input box (from google) and i haven't any other CSS in template for search bar (css is from google servers)

**Comment by mitar** [comment]
Strange that Google does not support that. Anyway, you have to set value of input box after Google code added it. So you have to do that in callback from it. Are you doing that?

**Comment by domenc** [comment]
I insert code in JS file (search_bar.js) after $('#google_results_link').prop('href', open_in_google_link);
and in HTML file after (all JS scripts) and doesnt work

**Comment by mitar** [comment]
Again. This is not the correct place. You have to call this code after Google code has done its thing. In callback from Google search.

**Comment by mitar** [comment]
Please finish this. You will also have to merge from current main repository.

**Comment by domenc** [comment]
All in issue #144

**Pull request closed without merging**